packages/ui/src/components/atoms/Input/InputLabel.tsx

```diff
@@ -1,32 +1,90 @@
-import React, { FC, ReactNode } from 'react';
-import styled from 'styled-components';
+import React, { FC, ReactNode, LabelHTMLAttributes } from 'react';
+import styled, { css } from 'styled-components';
 
-interface InputLabelProps {
+import {
+  spacing,
+  SpacingProps,
+  BorderProps,
+  border,
+  font,
+  display,
+  DisplayProps,
+  FontProps,
+} from '../../utils';
+
+interface InputLabelProps
+  extends LabelHTMLAttributes<HTMLLabelElement>,
+    SpacingProps,
+    BorderProps,
+    FontProps,
+    DisplayProps {
   children?: ReactNode;
+  className?: string;
+  fontSize?: number;
+  fontWeight?: FontProps['$fontWeight'];
+  textAlign?: 'left' | 'right';
+  noWrap?: boolean;
+  clickable?: boolean;
+  inline?: boolean;
+  color?: string;
 }
 
-const InputLabelStyle = styled.label`
-  display: block;
-  text-align: left;
+const InputLabelStyle = styled.label<InputLabelProps>`
+  ${font} // Utilize the font utility
+  ${display} // Utilize the display utility
```

Comment:
1. Utilities should be placed at the end.
2. Remove the comments

Reply:
done